src/backend/utils/misc/queryjumble.c

```diff
@@ -597,6 +597,24 @@ JumbleExpr(JumbleState *jstate, Node *node)
 				JumbleExpr(jstate, rtfunc->funcexpr);
 			}
 			break;
+		case T_Integer:
+			{
+				APP_JUMB(intVal(node));
+			}
+			break;
+		case T_GroupingFunc:
+			{
+				GroupingFunc *gf = (GroupingFunc *) node;
+				JumbleExpr(jstate, (Node *) gf->args);
+			}
+			break;
+		case T_GroupingClause:
+			{
+				GroupingClause *gc = (GroupingClause *) node;
+				APP_JUMB(gc->groupType);
+				JumbleExpr(jstate, (Node *) gc->groupsets);
+			}
+			break;
```

Comment:
the order of the cases in switch is mostly ordered by enum values, so I think it makes sense to swap T_GroupingFunc and T_GroupingClause in places.

Reply:
Ok, will swap